scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/crypto/Crypto.scala

```diff
@@ -275,3 +275,25 @@ case object PLAIN_TEXT extends CryptoMode {
    */
   override def secretKeyAlgorithm: String = ""
 }
+
+object Encrypter {
+
+  val COMMON = "common"
+  val NATIVE_AES_CBC = "nativeaescbc"
+
+  /**
+   * Create encrypter by string.
+   */
+   def apply(s: String): BigDLEncrypt = {
+     s.toLowerCase() match {
+       case COMMON =>
+         new BigDLEncrypt
+       case NATIVE_AES_CBC =>
+         new BigDLAESCBCEncrypt
+       case _ =>
+         throw new EncryptRuntimeException(
```

Comment:
use https://github.com/intel-analytics/BigDL/blob/dca1a0d179ffd986e3d31b4443fbd8cabcb7a63a/scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/utils/Log4Error.scala#L24 instead of throw exception

Reply:
we should use Log4Error's api instread of throw exception, see https://github.com/intel-analytics/BigDL/pull/4550.